tests/vanilla/subscribe.test.tsx

```diff
@@ -0,0 +1,123 @@
+import { describe, expect, it, vi } from 'vitest'
+import { subscribeWithSelector } from 'zustand/middleware'
+import { createStore } from 'zustand/vanilla'
+
+describe('subscribe()', () => {
```

Comment:
This is a nearly exact copy of `tests/subscribe.test.tsx`. Should I pull these tests into a utility function that can be reused with either `create` or `createStore`?

Reply:
As you ask, I think about it: I don't think we need to run comprehensive tests twice. We should probably run the comprehensive one for `createStore`, and for `create`, it could be a simple test to confirm that `subscribe` is properly attached to the created store. What do you think?

Reply:
Yeah, that seems reasonable, would you like to make this change or provide an explanation of what you'd like to see? Thank you

Reply:
I have an idea! I'll push my changes.